easybuild/easyconfigs/g/gdbgui/gdbgui-0.13.1.2-GCCcore-8.2.0-Python-3.7.2.eb

```diff
@@ -0,0 +1,96 @@
+        'source_urls': ['https://github.com/cs01/pygdbmi/archive/'],
+        'checksums': ['9c3b1757dd11d6d90a2bf47a22277f6c204646402950fa981daffe740c2493ad'],
+    }),
+    ('gevent', '1.4.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/g/gevent/'],
+        'checksums': ['1eb7fa3b9bd9174dfe9c3b59b7a09b768ecd496debfc4976a9530a3e15c990d1'],
+    }),
+    ('itsdangerous', '1.1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/i/itsdangerous/'],
+        'checksums': ['321b033d07f2a4136d3ec762eac9f16a10ccd60f53c0c91af90217ace7ba1f19'],
+    }),
+    ('Flask-Compress', '1.4.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/flask-compress/'],
+        'checksums': ['468693f4ddd11ac6a41bca4eb5f94b071b763256d54136f77957cfee635badb3'],
+    }),
+    ('Flask-SocketIO', '2.9.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/flask-socketio/'],
+        'checksums': ['f49edfd3a44458fbb9f7a04a57069ffc0c37f000495194f943a25d370436bb69'],
+    }),
+    ('Flask', '0.12.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/flask/'],
+        'checksums': ['2ea22336f6d388b4b242bc3abf8a01244a8aa3e236e7407469ef78c16ba355dd'],
+    }),
+    ('Click', '7.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/c/click/'],
+        'checksums': ['5b94b49521f6456670fdb30cd82a4eca9412788a93fa6dd6df72c94d5a8ff2d7'],
+    }),
+    ('Jinja2', '2.10.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/j/jinja2/'],
+        'checksums': ['065c4f02ebe7f7cf559e49ee5a95fb800a9e4528727aec6f24402a5374c65013'],
+    }),
+    ('Werkzeug', '0.15.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/w/werkzeug/'],
+        'checksums': ['a0b915f0815982fb2a09161cb8f31708052d0951c3ba433ccc5e1aa276507ca6'],
+    }),
+    ('socketio', '4.1.0', {
+        'source_tmpl': 'python-socketio-%(version)s.tar.gz',
+        'source_urls': ['https://pypi.python.org/packages/source/p/python-socketio/'],
+        'checksums': ['c7ffeac3d81f2d8d63b3ec7ed1e2d4478cc84aa1da666c1934c19432f4b8c0f8'],
+    }),
+    ('MarkupSafe', '1.1.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/m/markupsafe'],
+        'checksums': ['29872e92839765e546828bb7754a68c418d927cd064fd4708fab9fe9c8bb116b'],
+    }),
+    ('engineio', '3.8.1', {
+        'source_tmpl': 'python-engineio-%(version)s.tar.gz',
+        'source_urls': ['https://pypi.python.org/packages/source/p/python-engineio/'],
+        'checksums': ['9e4e7109d05d80ce5414f13b16f66725c2b5d574099fd43d37b024e7ea1c4354'],
+    }),
+    (name, version, {
+        'source_tmpl': '%(version)s.tar.gz',
+        'source_urls': ['https://github.com/cs01/gdbgui/archive/'],
+        'checksums': ['6d8eb296e664e4170ff063a3ad548bc7bd0badff04db59d825dcc5691e03730c'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/'],
+}
```

Comment:
these are the default `sanity_check_paths` for `PythonBundle`, so not necessary to add.